src/main/java/it/hospital/project/service/UserService.java

```diff
@@ -0,0 +1,19 @@
+package it.hospital.project.service;
+
+import it.hospital.project.appusers.User;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
```

Comment:
Do we plan to use somethere else UserService? Does it make sense to add additional level of abstraction?

Reply:
You are asking about this interface? Its easier to check all methods which userServiceImpl is implementing